src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -2541,6 +2553,55 @@ cfg_if! {
                     .finish()
             }
         }
+
+        // FIXME(msrv): `derive` on packed structs cannot be used below 1.67
```

Comment:
That sounds to me like a compelling reason to raise the main branch's MSRV.  Why is it currently set to 1.63?

Reply:
1.63 is what Debian stable currently ships so it is a pretty common baseline in the ecosystem. In a few months Trixie will be released 1.83 (or higher) so we will probably be able to do another bump (not that high, but at least above 1.63).